debian/changelog

```diff
@@ -1,9 +1,12 @@
-cloud-init (22.3-13-g70ce6442-0ubuntu1~22.10.2) UNRELEASED; urgency=medium
+cloud-init (22.4-0ubuntu1) lunar; urgency=medium
 
   * d/control: drop python3-httpretty from Build-Depends
   * d/cloud-init.templates: Add NWCS to datasource list
+  * Upstream snapshot based on 22.4 upstream release. List of changes
+    from upstream can be found at
+    https://raw.githubusercontent.com/canonical/cloud-init/main/ChangeLog
```

Comment:
I think we should point to the specific commit which also gives us a shorter URL for the win.
```suggestion
    https://github.com/canonical/cloud-init/blob/22.4/ChangeLog
```